app/src/ui/repository.tsx

```diff
@@ -84,11 +80,7 @@ export class RepositoryView extends React.Component<
       return null
     }
 
-    return enableNotificationOfBranchUpdates() ? (
-      <FilesChangedBadge filesChangedCount={filesChangedCount} />
-    ) : (
-      <Octicon className="indicator" symbol={OcticonSymbol.primitiveDot} />
-    )
+    return <FilesChangedBadge filesChangedCount={filesChangedCount} />
```

Comment:
Should this be behind beta flag or does it not matter since this is only targeting the beta release channel?

Reply:
I didn't think it was worth introducing a separate beta flag for just this feature, and I couldn't really find much context in the original PRs.

Reply:
> I couldn't really find much context in the original PRs about why we coupled it to NDDB.

It was originally part of NDDB because with the addition of the notification dot to indicate divergence, the meaning of the new changes notification dot was obscured. To fix, the dot was redesigned to be a badge that provides more useful context. It was de-scoped from the NDDB so I could focus on the MVP and to allow someone from the community to take it on.

Reply:
Aha, thanks for the context.

Yeah, I think we can ship this outside of the NDDB work.